employee.rb

```diff
@@ -0,0 +1,23 @@
+class Employee
+  def initialize(name:, salary:)
+    @name = name
+    @salary = salary
+    @reviews = []
+  end
+
+  def name
+    @name
+  end
+
+  def salary
+    @salary
+  end
+
+  def add_employee_review(review)
+    @review = review
```

Comment:
But you call it @review here.  I'd suggest deleting line 5, since you never use it.